tests/test_Apex.py

```diff
@@ -378,6 +378,26 @@ def test_convert_invalid_transformation():
         apex_out.convert(0, 0, 'geo', 'foobar')
 
 
+coord_names = ['geo', 'apex', 'qd']
+
+
+@pytest.mark.parametrize('transform', itertools.product(coord_names,
+                                                        coord_names))
+def test_convert_withnan(transform):
+    N_nans = 5
```

Comment:
Here, the docstring is missing and `N_nans` isn't snake case.  Again, fixing this myself.